docs/organizations/create-roles-permissions.mdx

```diff
@@ -1,11 +1,11 @@
 ---
-title: Create Roles and assign Permissions with Clerk
-description: Use the Clerk Dashboad to create and assign Roles and Permissions to users in your Organization.
+title: Create Roles and assign Permissions
+description: Use the Clerk Dashboard to create and assign Roles and Permissions to users in your Organization.
 ---
 
-# Create Roles and assign Permissions with Clerk
+# Create Roles and assign Permissions
 
-In the Clerk Dashboard, you can create Roles, assign Permissions to them, and change users' Roles.
+In the Clerk Dashboard, you can create Roles, assign Permissions to them, and change user's Roles.
```

Comment:
i think the previous syntax is also valid

Reply:
It is, it's a possessive plural